README.md

```diff
@@ -219,6 +219,17 @@ CPMAddPackage(
 )
 ```
 
+Bazel is also supported, simply add to your WORKSPACE file:
+```
+git_repository(
```

Comment:
[`http_archive` is preferred](https://docs.bazel.build/versions/3.5.1/external.html#repository-rules) so I would use that as the example.

Reply:
Thanks for the suggestion, done.